frontend/src/pages/performers/performerForm/PerformerForm.tsx

```diff
@@ -603,6 +616,16 @@ const PerformerForm: FC<PerformerProps> = ({
             ))}
           </Form.Control.Feedback>
 
+          <NavButtons onNext={() => setActiveTab("links")} />
+        </Tab>
+
+        <Tab eventKey="links" title="Links">
+          <Row>
+            <Col xs={8}>
+              <URLInput control={control} type={ValidSiteTypeEnum.PERFORMER} />
+            </Col>
+          </Row>
```

Comment:
The nav buttons are too far to the right,
and the rest is to match with the rest of the performer form.
```suggestion
        <Tab eventKey="links" title="Links" className="col-xl-9">
          <URLInput control={control} type={ValidSiteTypeEnum.PERFORMER} />
```